Lib/test/test_genericalias.py

```diff
@@ -293,5 +293,13 @@ def test_dir(self):
         for generic_alias_property in ("__origin__", "__args__", "__parameters__"):
             self.assertIn(generic_alias_property, dir_of_gen_alias)
 
+    def test_indexing(self):
+        with self.assertRaises(TypeError):
+            list[0]
```

Comment:
This doesn't check the other statements below and exits. Please consider using subtests : https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests

Reply:
Right, thanks for the helpful review and doc link! I thought it was okay to write it like that since the test_parameter_chaining function above it seems to not use subtests either (maybe that needs a PR some other time).

I've implemented the changes.